Src/Fido2/AuthenticatorAssertionResponse.cs

```diff
@@ -61,9 +62,10 @@ public static AuthenticatorAssertionResponse Parse(AuthenticatorAssertionRawResp
         uint storedSignatureCounter,
         IsUserHandleOwnerOfCredentialIdAsync isUserHandleOwnerOfCredId,
         IMetadataService? metadataService,
+        byte[]? requestTokenBindingId,
         CancellationToken cancellationToken = default)
     {
-        BaseVerify(config.FullyQualifiedOrigins, options.Challenge);
+        BaseVerify(config.FullyQualifiedOrigins, options.Challenge, requestTokenBindingId);
```

Comment:
Conformance requires requestTokenBinding to still be around, should be removed once the tool is updated.